lib/analytics/course_articles_csv_builder.rb

```diff
@@ -31,19 +31,28 @@ def set_articles_edited
     @course.all_revisions.includes(article: :wiki).map do |edit|
       article_edits = @articles_edited[edit.article_id] || new_article_entry(edit)
       article_edits[:characters][edit.mw_rev_id] = edit.characters
+      references_update(article_edits, edit)
```

Comment:
This should also be done with `edit.references_added` instead of reimplementing the calculation.

This is a confusing section of code, but it will be easier to follow if the assignment is done inline like with views and characters, like `article_edits[:references] = edit.references_added`.